chart/values.yaml

```
@@ -26,6 +26,14 @@ image:
   # -- ImagePullPolicy for our images
   pullPolicy: Always
 
+  # docker-secrets required to pull images if the container registry from image.Registry is protected
+  pullSecrets:
+    # -- Enable pullSecrets for pulling our container images
+    enabled: false
+    # -- Name of the pullSecret in the installed namespace
+    secrets:
+      - name: login
```

Looking at bitnami, they simply pass an array of secrets, presumably meaning its existence implies the enabled to true.
So I suggest we simply do the same:

```suggestion
  pullSecrets: []
```

---

fixed